libs/sdk-ui-semantic-search/src/hooks/useSemanticSearch.ts

```diff
@@ -161,3 +164,38 @@ const errorMessage = (e: unknown) => {
     }
     return String(e);
 };
+
+/**
+ * Enrich the result item with UI URL.
+ */
+const withUrl =
```

Comment:
I'm not sure about this. Search results should be clickable and lead to specific items. I did not find a good place where router links are defined. Is it the right place? Should it be in the gdc-ui instead?

Reply:
It would make sense to have it in gdc-ui instead but we don't have a place for that now.

Reply:
Possibly you can move this to `sdk-ui-ext` which is used for this purpose.